catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -91,6 +94,8 @@
 
     private List<String> fanoutProxyTagBlacklist = new ArrayList<>();
 
+    private long queryTimeout;
```

Comment:
Same comment about setting a default in the class to avoid potential initialization race conditions.